### PR TITLE
images/builder: let renovate update protoc and proto plugins

### PR DESCRIPTION
**Pull request by rolinh**
See commits for details.

### CODE REVIEW

File: images/builder/install-protoplugins.sh [hunk 1]
```diff
@@ -11,5 +11,5 @@ set -o nounset
 # 8ba23be9613c672d40ae261d2a1335d639bdd59b == tag: cmd/protoc-gen-go-grpc/v1.3.0
 go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@8ba23be9613c672d40ae261d2a1335d639bdd59b
 go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.31.0
-go install github.com/mitchellh/protoc-gen-go-json@49905733154f04e47d685de62c2cc2b72613b69e
```
Comment (aanm) [inline images/builder/install-protoplugins.sh:14]: Although it's a "blessed fork" it feels weird to use a dependency with only two/three stars.

**Review by aanm — APPROVED** [review]

**Review by gandro — APPROVED** [review]

File: images/builder/install-protoplugins.sh (hunk 1 above)
Comment (rolinh) [reply images/builder/install-protoplugins.sh:14]: Right but so far, the fork has been well maintained as opposed to the original repo which is officially unmaintained.

**Review by chancez — APPROVED** [review]

**Review by nebril — APPROVED** [review]

**Comment by gandro** [comment]
@rolinh It seems this PR broke the renovate config: https://github.com/cilium/cilium/issues/23230

File: .github/renovate.json5 [hunk 2]
```diff
@@ -420,8 +420,10 @@
     {
       "fileMatch": [
         "images/cilium/download-hubble\\.sh",
-        "images/runtime/build-gops.sh",
+        "images/builder/install-protoc.sh",
+        "images/builder/install-protoplugins.sh",
         "images/hubble-relay/download-grpc-health-probe.sh"
```
Comment (gandro) [inline .github/renovate.json5:425]: missing comma

Comment (rolinh) [reply .github/renovate.json5:425]: Oops, nice catch, will fix it!

Comment (rolinh) [reply .github/renovate.json5:425]: #32765

**Pull request merged**